src/util.ts

```diff
@@ -126,7 +126,11 @@ let telemtryReporter: TelemetryReporter;
 let toolsGopath: string;
 
 export function getGoConfig(): vscode.WorkspaceConfiguration {
-	return vscode.workspace.getConfiguration('go', vscode.window.activeTextEditor ? vscode.window.activeTextEditor.document.uri : null);
+	return vscode.window.activeTextEditor ? getGoConfigForUri(vscode.window.activeTextEditor.document.uri) : null;
```

Comment:
The code here has been tweaked to return null when there is no active text editor. Is this intentional? We need a valid go config to be returned at all times.